nifi-nar-bundles/nifi-lumberjack-bundle/nifi-lumberjack-processors/src/main/java/org/apache/nifi/processors/lumberjack/ListenLumberjack.java

```diff
@@ -0,0 +1,237 @@
+        "portion of one or more Lumberjack frames. In the case where the Lumberjack frames contain syslog messages, the " +
+        "output of this processor can be sent to a ParseSyslog processor for further processing.")
+@WritesAttributes({
+        @WritesAttribute(attribute="lumberjack.sender", description="The sending host of the messages."),
+        @WritesAttribute(attribute="lumberjack.port", description="The sending port the messages were received over."),
+        @WritesAttribute(attribute="lumberjack.sequencenumber", description="The sequence number of the message. Only included if <Batch Size> is 1."),
+        @WritesAttribute(attribute="lumberjack.*", description="The keys and respective values as sent by the lumberjack producer. Only included if <Batch Size> is 1."),
+        @WritesAttribute(attribute="mime.type", description="The mime.type of the content which is text/plain")
+})
+@SeeAlso(classNames = {"org.apache.nifi.processors.standard.ParseSyslog"})
+public class ListenLumberjack extends AbstractListenEventBatchingProcessor<LumberjackEvent> {
+
+    public static final PropertyDescriptor SSL_CONTEXT_SERVICE = new PropertyDescriptor.Builder()
+            .name("SSL Context Service")
+            .description("The Controller Service to use in order to obtain an SSL Context. If this property is set, " +
+                    "messages will be received over a secure connection. Note that as Lumberjack client requires"+
+                    "two-way SSL authentication, the controller MUST have a truststore and a keystore to work"+
+                    "properly.")
+            .required(false)
+            .identifiesControllerService(SSLContextService.class)
+            .build();
+
+    @Override
+    protected List<PropertyDescriptor> getAdditionalProperties() {
+        return Arrays.asList(
+                MAX_CONNECTIONS,
+                SSL_CONTEXT_SERVICE
+        );
+    }
+
+    @Override
+    protected Collection<ValidationResult> customValidate(final ValidationContext validationContext) {
+        final List<ValidationResult> results = new ArrayList<>();
+
+        final SSLContextService sslContextService = validationContext.getProperty(SSL_CONTEXT_SERVICE).asControllerService(SSLContextService.class);
+
+
+
+        if (sslContextService != null && sslContextService.isTrustStoreConfigured() == false) {
+            results.add(new ValidationResult.Builder()
+                    .explanation("The context service must have a truststore  configured for the lumberjack forwarder client to work correctly")
+                    .valid(false).subject(SSL_CONTEXT_SERVICE.getName()).build());
+        }
+
+        return results;
+    }
+
+    private volatile LumberjackEncoder lumberjackEncoder;
+    private volatile byte[] messageDemarcatorBytes; //it is only the array reference that is volatile - not the contents.
```

Comment:
I believe this is unused in favor of the delimiter from the superclass.  Is that correct?